src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -169,7 +182,9 @@ export default {
         },
         attrs: {
           'aria-colindex': String(colIndex + 1),
-          ...this.getTdValues(item, key, field.tdAttr, {})
+          ...(field.isRowHeader
+            ? this.getThValues(item, key, field.thAttr, 'body', {})
```

Comment:
The type here should be `row`, rather than `body`, to match with other callbacks on tbody rows